scala_proto/scala_proto.bzl

```diff
@@ -351,7 +353,7 @@ def _gen_proto_srcjar_impl(ctx):
         # Command line args to worker cannot be empty so using padding
         flags_arg = "-" + ",".join(ctx.attr.flags),
         # Command line args to worker cannot be empty so using padding
-        packages = "-" + ":".join(depset(transitive = transitive_proto_paths).to_list())
+        packages = "-" + ":".join(sorted(depset(transitive = transitive_proto_paths).to_list()))
```

Comment:
Strange that our reproducibility test didn’t catch it

Reply:
Depsets are supposed to be in a consistent order. This is a bummer that we also need to sort.

Reply:
I'm not quite sure how we can exactly hit this case in our tests, but from what i can see the input hash/args change which stops it being a cache hit. but the resulting jar is identical so our reproducibility tests wouldn't see anything wrong